plumbing/dag/dag.go

```diff
@@ -70,3 +71,17 @@ func (dag *DAG) GetFileSize(ctx context.Context, c cid.Cid) (uint64, error) {
 		return 0, fmt.Errorf("unrecognized node type: %T", fnode)
 	}
 }
+
+// Cat returns an iostream with a piece of data stored on the merkeldag with
+// the given cid.
+func (dag *DAG) Cat(ctx context.Context, c cid.Cid) (io.DagReader, error) {
+	// TODO: this goes back to 'how is data stored and referenced'
```

Comment:
Not your TODO but is this reflected in an issue in go-filecoin?  I'm guessing yes but would be good to confirm and link the issue here, and if not write the issue and then link here.

I don't know exactly what the TODO is talking about. If you're in the same boat it's worth git blaming and badgering the author for their meaning / any issues they know about.

Reply:
yeah this is almost certainly why, and you should ask him whether it is ok to remove this todo. if we need a todo then we should have an associated issue that it can point to.